main.go

```diff
@@ -114,7 +119,7 @@ func before(c *cli.Context) error {
 		return err
 	}
 
-	client = container.NewClient(!c.GlobalBool("no-pull"))
+	client = container.NewClient(!c.GlobalBool("no-pull"), !c.GlobalBool("no-watch"))
```

Comment:
The command line option is "no-enable" while this looks for "no-watch". In fact I'd probably prefer a slightly more verbose option name like "no-watch-by-default"